### ISSUE TITLE
drawEdge multiple edges

### ISSUE THREAD

**Issue by kakila** [mattools/matGeom#71]
The following snippet when run in GNU Octave 5.1.0

```octave
E = randn (10, 4);
h = drawEdge(E);
```
produces and array `h` with 10 handles, but only one is valid (the last one), and a single edge is plotted. This is because drawEdge at no point sets the hold state of the axes.

Is this also the case in matlab?
Does matlab holds the contents of the axes in the following example?

```matlab
plot (rand(10,1), 'o');
ax = gca ();
plot(ax, rand(10,1), 'x')
```

In Octave you only see the crosses.

**Comment by dlegland** [comment]
Hi,
yes, same behaviour in Matlab.

The drawXXX currently follow the plot behaviour of Matlab (-> the hold state is not changed by the functions), and some functions may have the same problem (showing only the last item).
I'll fix drawEdge.

**Comment by dlegland** [comment]
In your second example, Matlab also shows only the crosses.

**Comment by kakila** [comment]
Hi,

It is good that the functinos do not change the hold state, and it is also very useful that the functions add several objects to the axes. The following biolerplate should do the trick

```octave
onhold = ishold();
hold on

<ploting code here>

if ~onhold
   hold off
end
```

**Comment by dlegland** [comment]
Fixed in 601460db0ce6fb5b6fc8df31e8be1896c659da88
